app/views/spree/orders/form/_update_buttons.html.haml

```diff
@@ -1,12 +1,19 @@
 .row
-  .columns.small-12.medium-3
+  .columns.small-12.medium-5
     - if current_order.nil? || current_order.distributor.nil? || current_order.distributor == @order.distributor
       - if current_order&.line_items.present?
         = link_to main_app.cart_path, :class => "button expand" do
           = t(:order_back_to_cart)
       - else
-        = link_to "#{main_app.enterprise_shop_path(@order.distributor)}#/shop", class: "button expand" do
-          = t(:order_back_to_store)
+      .wrapper
+        .row
+          .columns.small-12.medium-6
+            = link_to "#{main_app.enterprise_shop_path(@order.distributor)}#/shop", class: "button expand" do
+              = t(:order_back_to_store)
+          .columns.small-12.medium-6
+            - if @order.distributor.website.present?
+              = link_to_service "http://", current_order.distributor.website, class: "button expand" do
+                = t(:order_back_to_website)
```

Comment:
What is the reasoning for opening a link with http here? For our customers sake, we shouldn't be opening any unsecured websites imo